money/currency.py

```diff
@@ -1073,7 +1073,7 @@ class CurrencyHelper:
             'display_name': 'New Taiwan Dollar',
             'numeric_code': 901,
             'default_fraction_digits': 2,
-            'sub_unit': 100,
```

Comment:
For this one, the lists disagree and wikipedia says the subunit is "rarely" used